<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Duchar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStack">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="stack">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <layout class="QVBoxLayout" name="duqf_settingsLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="titleWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>13</number>
           </property>
           <property name="topMargin">
            <number>3</number>
           </property>
           <property name="rightMargin">
            <number>3</number>
           </property>
           <property name="bottomMargin">
            <number>3</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="duqf_reinitSettingsButton">
             <property name="toolTip">
              <string>Reset all settings to their default values and restart the application</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="duqf_resources.qrc">
               <normaloff>:/icons/reinit</normaloff>:/icons/reinit</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="duqf_closeSettingsButton">
             <property name="toolTip">
              <string>Close settings</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="duqf_resources.qrc">
               <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="buttonListPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QListWidget" name="buttonList">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::InternalMove</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>15</width>
     <height>15</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClipboard_History"/>
   <addaction name="actionManage_buttons"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
   <addaction name="actionAdd_to_favorites"/>
   <addaction name="actionRemove"/>
   <addaction name="actionShow_Hide"/>
   <addaction name="actionQuit"/>
  </widget>
  <widget class="QStatusBar" name="mainStatusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="duqf_resources.qrc">
     <normaloff>:/icons/close</normaloff>:/icons/close</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionManage_buttons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/fav</normaloff>
     <normalon>:/icons/fav_l</normalon>:/icons/fav</iconset>
   </property>
   <property name="text">
    <string>Manage favorites...</string>
   </property>
   <property name="toolTip">
    <string>Favorites</string>
   </property>
  </action>
  <action name="actionShow_Hide">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/hide</normaloff>:/icons/hide</iconset>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
   <property name="toolTip">
    <string>Show or hide application</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/trash</normaloff>:/icons/trash</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear the copy history</string>
   </property>
  </action>
  <action name="actionAdd_to_favorites">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/add</normaloff>:/icons/add</iconset>
   </property>
   <property name="text">
    <string>Add to favorites</string>
   </property>
   <property name="toolTip">
    <string>Add to favorites</string>
   </property>
  </action>
  <action name="actionClipboard_History">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/clipboard</normaloff>
     <normalon>:/icons/clipboard_l</normalon>:/icons/clipboard</iconset>
   </property>
   <property name="text">
    <string>Clipboard History</string>
   </property>
   <property name="toolTip">
    <string>Clipboard History</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/minus</normaloff>:/icons/minus</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="duqf_resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
